### PR TITLE
Create blog post for GPS outage

### PR DESCRIPTION
**Pull request by DenisePeterson**
Sending out a notice to the GPS network users that there could be an issue.

### CODE REVIEW

File: _posts/2023-09-13-possible-gps-disruption-from-sept-13-29.md [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+categories:
+  - SGID Blog
+tags:
+  - turngps
+  - gps
+  - gps_network
+  - gps_disruption
+---
+<p>
+Utah Geospatial Resource Center (UGRC), along with the Utah Statewide Information and Analysis Center (SIAC), has been made aware of planned GPS testing that will lead to intermittent GPS disruptions throughout large portions of Utah, Idaho, and a small portion of Wyoming between September 13th - 29th, 2023. The testing will be centered at both Dugway Proving Grounds, Utah and near Mountain Home, Idaho.
```
Comment (stdavis) [inline _posts/2023-09-13-possible-gps-disruption-from-sept-13-29.md:16]:
```suggestion
The Utah Geospatial Resource Center (UGRC), along with the Utah Statewide Information and Analysis Center (SIAC), has been made aware of planned GPS testing that will lead to intermittent GPS disruptions throughout large portions of Utah, Idaho, and a small portion of Wyoming between September 13th - 29th, 2023. The testing will be centered at both Dugway Proving Grounds, Utah and near Mountain Home, Idaho.
```

File: _posts/2023-09-13-possible-gps-disruption-from-sept-13-29.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    </tr>
+    <tr>
+      <td>29 SEP 23</td>         
+      <td>1900 MDT - 2300 MDT</td>
+    </tr>
+  </table>
+</p>
+<p>
+Contact UGRC at the Support Desk if you experience GPS disruptions during this time.
+Support Desk at 801-538-3440 or toll-free 1-800-678-3440.
```
Comment (stdavis) [inline _posts/2023-09-13-possible-gps-disruption-from-sept-13-29.md:139]:
```suggestion
Contact UGRC at the Support Desk if you experience GPS disruptions during this time.
The Support Desk can be reached at 801-538-3440 or toll-free at 1-800-678-3440.
```

**Review by stdavis — CHANGES_REQUESTED** [review]
This would have been way less typing if you used Markdown. I'm happy to give you an introduction to it if you are not familiar with it. But, other than a few nitpicks, I think this looks great.

Do you want to add the "Featured" category so that it shows up in the header on the home page?

**Comment by steveoh** [comment]
She mentioned they had trouble with markdown so I converted it to see if there is an issue. @DenisePeterson Please [read the readme](https://github.com/agrc/gis.utah.gov/blob/master/README.md#contact-information) as we don't put phone numbers or emails directly in blog posts. This allows us to change the value in one place instead of however many places it is referenced.

**Comment by steveoh** [comment]
I don't see any issues with the markdown. Figured you might want to get this out quickly for the urgency factor.

**Pull request merged**